### PR TITLE
source-*: switch all source connectors to TCP

### PR DESCRIPTION
**Pull request by mdibaiee**
**Description:**

- migrate all source connectors to use TCP

**Workflow steps:**

- as usual!

**Documentation links affected:**

(list any [documentation links](https://docs.google.com/document/d/1SRC9VS9zyCzWl3n4HXHbc4wPB1eLxJHkA2rtu9ZNokM/edit?usp=sharing) that you created, or existing ones that you've identified as needing updates, along with a brief description)

**Notes for reviewers:**

(anything that might help someone review this PR)

- - -
This change is [Reviewable](https://reviewable.io/reviews/estuary/connectors/378)

### CODE REVIEW

File: source-boilerplate/boilerplate.go [hunk 1]
```diff
@@ -25,11 +26,24 @@ func RunMain(srv pc.DriverServer) {
 	var parser = flags.NewParser(nil, flags.Default)
 	var ctx, _ = signal.NotifyContext(context.Background(), syscall.SIGTERM, syscall.SIGINT)
 
+	var conn net.Conn
+	if server, err := net.Listen("tcp", ":2222"); err != nil {
+		logrus.WithFields(logrus.Fields{"error": err}).
+			Error("failed to start tcp server")
+		os.Exit(1)
+	} else if conn, err = server.Accept(); err != nil {
+		if err != nil {
```
Comment (willdonnelly) [inline source-boilerplate/boilerplate.go:35]: nit: Duplicate `if err != nil` check?

File: source-boilerplate/boilerplate.go [hunk 2]
```diff
@@ -25,11 +26,24 @@ func RunMain(srv pc.DriverServer) {
 	var parser = flags.NewParser(nil, flags.Default)
 	var ctx, _ = signal.NotifyContext(context.Background(), syscall.SIGTERM, syscall.SIGINT)
 
+	var conn net.Conn
+	if server, err := net.Listen("tcp", ":2222"); err != nil {
+		logrus.WithFields(logrus.Fields{"error": err}).
+			Error("failed to start tcp server")
```
Comment (willdonnelly) [inline source-boilerplate/boilerplate.go:32]: You can use `logrus.Fatal()` here, it's basically just a helper for `logrus.Error(); os.Exit(1)` already.

**Review by willdonnelly — APPROVED** [review]
LGTM

**Comment by mdibaiee** [comment]
I actually still have to add network-tunnel to the source connectors that have `address` field, almost forgot about that

**Comment by mdibaiee** [comment]
Tested source-postgres and source-mysql's new network-tunnel integration and they both look okay. I bumped both versions because of the change in endpoint_spec_schema.

**Pull request merged**